test/unit/home-content.test.js

```diff
@@ -0,0 +1,76 @@
+const homeContent = require('../mocks/mockHomeContent')
```

Comment:
Your reference is to mockHomeContent, hence, you are actually testing mockHomeContent but not homeContent. The intention is to test homeContent.

Reply:
I have tweaked the test file to include checks for the rendered page as discussed